src/EfuncMsg.chpl

```diff
@@ -81,6 +82,25 @@ module EfuncMsg
                         var a = st.addEntry(rname, e.size, real);
                         a.a = Math.cos(e.a);
                     }
+                    when "hash64" {
```

Comment:
Could you explain the benefit of using `enum` for the subcommand name?

Reply:
enforcing controlled vocabulary as opposed to free-form strings where the developer could mistype something and only see the error at runtime.

Reply:
But in this case, the subcommand name is not a free-form string that the developer types in; rather, it comes from the client-generated message. True, the developer must type it in on the client side, but using an `enum` in the server wouldn't catch that kind of typo anyway, because there's no way for the server to pre-validate client-generated messages at compile time. 

If these subcommands were reused elsewhere in the server code, then I could see using an `enum` with a converter function that takes a string and returns one of the `enum` options (like we do with `DType`), but these subcommands are single-use, so I don't think an `enum` would actually decrease the opportunity for error.